src/sbt-test/proguard/filter/build.sbt

```diff
@@ -1,16 +1,16 @@
-enablePlugins(SbtProguard)
-
-scalaVersion := "2.12.3"
+import java.nio.file.FileSystems
 
-proguardOptions in Proguard += "-dontoptimize"
-
-proguardOptions in Proguard ++= Seq("-dontnote", "-dontwarn", "-ignorewarnings")
+enablePlugins(SbtProguard)
 
-proguardOptions in Proguard += ProguardOptions.keepMain("Test")
+scalaVersion := "2.13.6"
+name := "filter"
 
-proguardInputFilter in Proguard := { file =>
+(Proguard / proguardMerge) := true
+(Proguard / proguardOptions) ++= Seq("-dontoptimize", "-dontnote", "-dontwarn", "-ignorewarnings")
+(Proguard / proguardOptions) += ProguardOptions.keepMain("Test")
+(Proguard / proguardInputFilter) := { file =>
   file.name match {
-    case "scala-library.jar" => Some("!META-INF/**")
+    case "scala-library-2.13.6.jar" => Some("!META-INF/**")
```

Comment:
could this be based off `scalaVersion` instead of being hardcoded? otherwise I fear we'll forget to update it; doubt Scala Steward would pick this up

Reply:
I actually tried that originally, but I don't think we can use interpolated strings in pattern-matches. It kept giving me errors

Reply:
how about just `if (file.name == s"scala-library-{scalaVersion.value}.jar") Some(...) else None` rather than a pattern match? It's possible to do it with pattern matching but it seems like overkill.

Reply:
+1 what Seth said

I added the missing $, filled the blanks and pushed it to https://github.com/mpollmeier/sbt-proguard/tree/sbt-1.5
`sbt scripted` is still green.

Reply:
OK, sorry I had a brainfart, I merged @mpollmeier 's changes